packages/neuron-ui/src/containers/Providers/index.tsx

```diff
@@ -89,11 +91,39 @@ const withProviders = (Comp: React.ComponentType) => (props: React.Props<any>) =
     },
   )
 
+  UILayer.on(Channel.GetWalletNameListStore, (_e: Event, args: Response<any>) => {
+    if (args.status) {
+      setWalletNames({
+        name: args.result,
+      })
+    }
+  })
+
+  UILayer.on(Channel.SaveWalletStore, (_e: Event, args: Response<any>) => {
```

Comment:
As above, these interfaces should pass a wallet object. So better to have `SaveWallet`, `DeleteWallet` etc.